internal/sdkv2provider/provider.go

```diff
@@ -276,6 +276,7 @@ func New(version string) func() *schema.Provider {
 				"cloudflare_observatory_scheduled_test":                      resourceCloudflareObservatoryScheduledTest(),
 				"cloudflare_origin_ca_certificate":                           resourceCloudflareOriginCACertificate(),
 				"cloudflare_page_rule":                                       resourceCloudflarePageRule(),
+				"cloudflare_page_shield":                                     resourceCloudflarePageShield(),
```

Comment:
Willing to change the name to: `cloudflare_page_shield_settings`

Reply:
we should definitely rename this resource to be more specific however, we also aren't accepting more resources added to the SDKv2 so we'll need to port this into the `terraform-plugin-framework` instead. we have [a ratchet](https://github.com/cloudflare/terraform-provider-cloudflare/pull/4007/files#diff-4ae7bd7759a37ef1ead791fcae9d7d3a02156d7f9fd95c7c614f3067e8a132bfR23) in place that notifies you when you try and run the acceptance test suite (acc tests won't run on PRs due to creating real resources and taking forever to run).